Geometry/TrackerCommonData/python/cmsExtendedGeometry2021ZeroMaterialXML_cfi.py

```diff
@@ -0,0 +1,380 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Is this file generated? If not, why not? You added the capability to do so in #31082.

Reply:
Or maybe the incorrect comment in the other PR has me confused.